### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by matiasvlevi** [matiasvlevi/Dann#7] [General]
# Contribution
Welcome contributors!

Ask any questions regarding contributions in this discussion section. 

Acces the contribution docs [here](https://github.com/matiasvlevi/Dann/blob/master/CONTRIBUTING.md)

**Comment by robonetphy** [comment]
Hey there guys, I want to contribute to this community and I have gone throw the contribute.md also. Can you guide me with some `beginner issues` or `bug` through which I can start contributing....

**Reply by matiasvlevi** [reply]
Sure! Feel free to join our discord server, it makes communication much easier. 
We can walk you through the contributing workflow.
[Invite here](https://discord.gg/yeAqHsGdtU)

Also, have you taken a look at the issues tab? These are the current goals for contributions, you could also submit one.

**Reply by robonetphy** [reply]
@matiasvlevi Thanks for the reply, I join the discord server. Also, I want to start with something so if you can assign something to me... or give me some issue which will kinda give more insight about the library, which will be helpful.

**Reply by matiasvlevi** [reply]
Great! ill follow up with you on the discord server.
